applications/fastapi-bootcamp/values-idfdemo.yaml

```diff
@@ -0,0 +1,6 @@
+image:
+  pullPolicy: Always
+  tag: "tickets-DM-43967"
```

Comment:
Recommend deleting to use the appVersion instead:

```suggestion
  tag: "tickets-DM-43967"
```

Reply:
See above.